src/constants/data.ts

```diff
@@ -1,8 +1,10 @@
 import { easy, hard, CorrectEntriesIcon, IncorrectEntriesIcon, AlmostCorrectEntriesIcon } from '../../assets/images'
 import { RoutesParamsType } from '../navigation/NavigationTypes'
 
+export type ExerciseKeyType = 'vocabularyOverview' | 'vocabularyTrainer'
```

Comment:
Kannst du hier bitte den union type so ändern, wie in dem issue beschrieben:

https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail&selectedIssue=LUN-57

Reply:
Hmm why if there is an own issue for this? :see_no_evil:

Reply:
Thoght because you created a new type, so why not do it the correct way in the first place. Or this is an new type from you, isn't it?